src/status_im/ui/screens/chat/message/link_preview.cljs

```diff
@@ -19,9 +19,14 @@
     (string/starts-with? link (str "https://www." domain)) true
     :else                                                  false))
 
+;;siddarthkay : todo add code for previews here once previews are fixed in new messaging ui
+
 (defn community-id-from-link
   [link]
-  (nth (re-find constants/regx-community-universal-link link) 4))
+  (let [community-id (nth (re-find constants/regx-community-universal-link link) 4)]
```

Comment:
Why the changes in this function? It seems to unnecessarily store in the community ID in a binding.

Reply:
hmm yeah I will revert the changes here, I was doing some work here but then decided to revert it, which left behind this code block.